### PR TITLE
feat(client): ts-migrate (/client/src/components/layouts/Certification.js)

### PR DESCRIPTION
**Pull request by RaymenDeol**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Ref #42256

### CODE REVIEW

File: client/src/components/layouts/certification.tsx [hunk 1]
```diff
@@ -1,47 +1,48 @@
...
+}
+
 const mapStateToProps = createSelector(isSignedInSelector, isSignedIn => ({
   isSignedIn
 }));
 
 const mapDispatchToProps = { fetchUser, executeGA };
 
-class CertificationLayout extends Component {
+class CertificationLayout extends Component <CertificationProps> {
```
Comment (nhcarrigan) [inline client/src/components/layouts/certification.tsx:22]:
```suggestion
class CertificationLayout extends Component<CertificationProps> {
```
It's nitpicky, but this is what the linter wants. :)

File: client/src/components/layouts/certification.tsx [hunk 2]
```diff
@@ -1,47 +1,48 @@
...
+    }
   }
 
-  render() {
+  render(): JSX.Element {
     const { children } = this.props;
 
     return (
-      <>
+      <Fragment>
```
Comment (ojeytonwilliams) [inline client/src/components/layouts/certification.tsx:38]:
```suggestion
      <>
```
We recently changed the eslint rules to require the shorthand: https://github.com/freeCodeCamp/freeCodeCamp/pull/42963/

File: client/src/components/layouts/certification.tsx [hunk 3]
```diff
@@ -1,47 +1,48 @@
...
         <Helmet bodyAttributes={{ class: 'light-palette' }} />
         {children}
-      </>
+      </Fragment>
```
Comment (ojeytonwilliams) [inline client/src/components/layouts/certification.tsx:41]:
```suggestion
      </>
```

**Review by ojeytonwilliams — DISMISSED** [review]
Another linting issue, I'm afraid, but aside from that this looks good.

File: client/src/components/layouts/certification.tsx [hunk 4]
```diff
@@ -1,47 +1,48 @@
-import React, { Component } from 'react';
-import PropTypes from 'prop-types';
+import React, { Fragment, Component } from 'react';
```
Comment (nhcarrigan) [inline client/src/components/layouts/certification.tsx:1]:
```suggestion
import React, { Component } from 'react';
```
In combination with Oliver's suggestion, we'll need to remove the unused import.

**Review by naomi-lgbt — DISMISSED** [review]

**Review by ShaunSHamilton — APPROVED** [review]
LGTM 👍

**Review by ojeytonwilliams — APPROVED** [review]
LGTM :+1:

**Comment by ojeytonwilliams** [comment]
@RaymenDeol Congrats on your first pull request (PR)! 🎉

Thank you for your contribution to the codebase! 👍
We're happy to accept these changes, and look forward to future contributions. 📝

**Pull request merged**